package.json

```diff
@@ -29,16 +30,21 @@
 		"@googlemaps/google-maps-services-js": "^3.1.16",
 		"@graphql-tools/graphql-file-loader": "^6.2.7",
 		"@graphql-tools/load": "^6.2.8",
+		"@graphql-tools/schema": "^7.1.3",
 		"apollo-server": "^2.22.2",
 		"graphql": "^15.5.0",
 		"opencage-api-client": "^1.0.0",
 		"word2vec": "^1.1.4",
 		"yelp-fusion": "^3.0.0"
 	},
 	"devDependencies": {
+		"@graphql-codegen/cli": "1.21.3",
+		"@graphql-codegen/typescript": "1.21.1",
 		"@types/node": "^14.14.37",
 		"concurrently": "^6.0.1",
 		"nodemon": "^2.0.7",
-		"typescript": "^4.2.4"
+		"typescript": "^4.2.4",
+		"@graphql-codegen/typescript-resolvers": "1.19.0",
```

Comment:
The ordering of the devDependencies seems off

Reply:
Yes, that is because a different tool (`graphql-codegen` instead of `npm` added them. I don't actually need those dependencies anymore, though.